ivy/functional/frontends/jax/func_wrapper.py

```diff
@@ -53,6 +57,48 @@ def _to_ivy_array(x):
     return _from_jax_frontend_array_to_ivy_array(_native_to_ivy_array(x))
 
 
+# API Functions #
+# ------------- #
+
```

Comment:
Turns out `API Functions` does not fit all cases after all. `Main` would be more general and definitely applicable everywhere. Also, I think it would stand out more if we added `---` on the first header line as well.
Could you please update to the following? Thanks a lot!
```
# --- Helpers --- #
# --------------- #
```
```
# --- Main --- #
# ------------ #
```